md/rest_extensions.md

```diff
@@ -19,18 +22,29 @@ Get list of running cases
 |search|string|false|Search keyword, will search in case id and name of the initiator|
 
 ## `/bici-process-api`
```

Comment:
in the table: "Page to return, starts at 0"
                    " late, predictedLate, onTime"